packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -525,8 +521,6 @@ export function attemptHydrationAtCurrentPriority(fiber: Fiber): void {
   markRetryLaneIfNotHydrated(fiber, lane);
 }
 
-export {getCurrentUpdatePriority, runWithPriority};
```

Comment:
couldn't figure out why this was being exported. nothing breaks in tests but maybe it's important to keep?